tooling/ssa_fuzzer/README.md

```diff
@@ -0,0 +1,33 @@
+# SSA Fuzzer
+
+A fuzzing tool for testing and comparing ACIR and Brillig implementations based on [cargo-fuzz](https://github.com/rust-fuzz/cargo-fuzz).
+
+## Overview
+
+This fuzzer generates random sequences of arithmetic and logical operations to craft SSA from scratch and than verify that both ACIR and Brillig implementations produce identical results. It helps catch potential bugs and inconsistencies between the two implementations.
+
+
+## Usage
+
+1. Setup environment:
+You need to delete `rust-toolchain.toml` from root of the project, because cargo-fuzz requires nightly compiler for address sanitizer.
+```
+rustup install nightly
+rustup default nightly
```

Comment:
I found that this command works as well, so I don't have to default to `nightly`:

```
cargo +nightly fuzz ...
```